.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 build
-zkeyFiles
\ No newline at end of file
+zkeyFiles
+tau
```

Comment:
As you mentioned, IMO we should add `compiled` to `.gitignore` until we settle down to a ceremony